milestone-3/backend/index.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express')
+const app = express()
+app.use(express.json())
+const mongoose = require('mongoose')
+const connection_url = "mongodb+srv://newUser:<newPassword>@websitecluster.syy0q.mongodb.net/RecipesDB?retryWrites=true&w=majority"
+
+mongoose.connect(connection_url)
+.then(() => console.log('Successfully connected'))
+.catch((error) => console.error(`Could not connect due to ${error}`))
+
+const Recipe = new mongoose.Schema({
+    recipe_name: String,
+    recipe_description: String
+}, {collection: 'Recipes'})

```

Comment:
A few notes on this schema:
To help keep track of naming I may suggest using RecipeSchema and Recipe instead of Recipe and Setup respectively. Setup is a bit harder to follow in the code since it is not super descriptive.

When setting up schemas, it helps to think about what we want our actual documents to look like. Here you have defined the name and description fields in a document, but there should be other data we need to keep track of with recipes. Things such as ingredient list and an instruction list. Make sure all the data you want to store is represented in your schema.

Finally, this is just a small comment, but you should stick with camel case or snake case throughout your code when naming variables. Just something that will help with readability in project teams.